test/jasmine/tests/decode_typed_arrays_test.js

```diff
@@ -0,0 +1,160 @@
+var Plotly = require('@lib/index');
+var b64 = require('base64-arraybuffer');
+var encodedFigure = {
+    'data': [{
+        'type': 'scatter',
+        'x': {'dtype': 'float64', 'value': 'AAAAAAAACEAAAAAAAAAAQAAAAAAAAPA/'},
+        'y': {'dtype': 'float32', 'value': 'AABAQAAAAEAAAIA/'},
+        'marker': {
+            'color': {
+                'dtype': 'uint16',
+                'value': 'AwACAAEA',
+            },
+        }
+    }]
+};
+
+var typedArraySpecs = [
+    ['int8', new Int8Array([-128, -34, 1, 127])],
+    ['uint8', new Uint8Array([0, 1, 127, 255])],
+    ['uint8_clamped', new Uint8ClampedArray([0, 1, 127, 255])],
+    ['int16', new Int16Array([-32768, -123, 345, 32767])],
+    ['uint16', new Uint16Array([0, 345, 32767, 65535])],
+    ['int32', new Int32Array([-2147483648, -123, 345, 32767, 2147483647])],
+    ['uint32', new Uint32Array([0, 345, 32767, 4294967295])],
+    ['float32', new Float32Array([1.2E-38, -2345.25, 2.7182818, 3.1415926, 2, 3.4E38])],
+    ['float64', new Float64Array([5.0E-324, 2.718281828459045, 3.141592653589793, 1.8E308])]
```

Comment:
It might be of interest to add `BigInt`s to the list.

Reply:
Seems that is not yet supporting longs. How is ploty dealing with BigInt currently? is there any support planned?